src/Traits/LogsActivity.php

```diff
@@ -316,9 +316,9 @@ function ($new, $old) {
                     // Handels Date intervels comparsons since php cannot use spaceship
                     // Operator to compare them and will throw ErrorException.
                     if ($old instanceof DateInterval) {
-                        return CarbonInterval::make($old)->equalTo($new);
+                        return intval(! CarbonInterval::make($old)->equalTo($new));
```

Comment:
Could you use something like `CarbonInterval::make($old)->equalTo($new) ? 0 : 1` - the current one is correct but not really readable.